modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts

```diff
@@ -124,6 +113,34 @@ const register = (editor: Editor): void => {
   });
 };
 
+const calcCols = (editor: Editor, colors: number): number => {
+  // Sensible default based on the amount of colors in 'color_map'
+  const defaultCols = Math.ceil(Math.sqrt(option('color_map')(editor).length));
+  return Math.max(defaultCols, Math.ceil(Math.sqrt(colors)));
```

Comment:
This was necessary for calculating from the color_map_* options, just refactored this. An outdated test was failing, so I removed it since it was unnecessary with the new scope of testing for these options.